src/test/java/com/customer/customerService/CrediCardApplicationTest.java

```diff
@@ -0,0 +1,166 @@
+package com.customer.customerService;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.customer.model.CreditCard;
+import com.customer.repository.CreditCardRepository;
+import com.customer.service.CreditCardServiceImpl;
+
+
+
+public class CrediCardApplicationTest {
+	
+	CreditCard creditCardData, creditCardDataWithNull, creditCardData1, creditCardData2, creditCardData3, creditCardData4;
+	
+	List<CreditCard> creditCardList;
+	
+	CreditCardRepository creditCardRepositoryMock;
+	
+	CreditCardServiceImpl creditCardServiceImpl;
+	
+	List<CreditCard> creditCardList1;
```

Comment:
Try to use ImmutableList for tests... read more for the reason why

Reply:
Done.